src/ts/edit.ts

```diff
@@ -249,6 +249,28 @@ document.addEventListener('DOMContentLoaded', async () => {
       }
       editor.setContent(content);
 
+    // INSERT VIDEO AT CURSOR POSITION IN TEXT
+    } else if (el.matches('[data-action="insert-video-in-body"]')) {
+      // link to the video
+      const url = `app/download.php?name=${encodeURIComponent(el.dataset.name)}&f=${encodeURIComponent(el.dataset.link)}&storage=${encodeURIComponent(el.dataset.storage)}`;
+      // no syntax for video in markdown; use plain html in both cases
+      const video = document.createElement('video');
+      const source = document.createElement('source');
+      source.src = url;
+      video.setAttribute('controls', 'controls');
+      video.appendChild(source);
```

Comment:
what about making it a bit wider by default? currently 300px width, doubling that should do the trick.